Makefile

```diff
@@ -124,6 +128,31 @@ e2e-helm-deploy:
 		./.staging/helm/linux-amd64/helm install manifest_staging/charts/gatekeeper --name-template=gatekeeper --set image.repository=${HELM_REPO} --set image.release=${HELM_RELEASE} --set emitAdmissionEvents=true --set emitAuditEvents=true;\
 	fi;
 
+e2e-helm-upgrade-init: e2e-helm-install
+	@if [ $$(echo ${HELM_VERSION} | head -c 1) = "2" ]; then\
+		kubectl create clusterrolebinding tiller-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default;\
+		./.staging/helm/linux-amd64/helm init --wait --history-max=5;\
+		kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s;\
+		./.staging/helm/linux-amd64/helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts;\
```

Comment:
Is this part of the branch and the next branch the same code? Could we put the common code after the if-statement?

Reply:
they are slightly different `--name-template` vs `name` (helm 3 vs 2)

Reply:
makes sense, thanks!